.github/config/labeler-config.yml

```diff
@@ -1,40 +1,44 @@
 # Pull Request Labeler Github Action Configuration: https://github.com/marketplace/actions/labeler
 "tests:hive":
-  - lib/trino-orc/**
-  - lib/trino-parquet/**
-  - lib/trino-hive-formats/**
-  - plugin/trino-hive/**
-  - testing/trino-product-tests/**
-  - lib/trino-filesystem/**
-  - lib/trino-filesystem-*/**
+  - changed-files:
+    - any-glob-to-any-file: ['lib/trino-orc/**', 'lib/trino-parquet/**', 'lib/trino-hive-formats/**', 'plugin/trino-hive/**', 'testing/trino-product-tests/**', 'lib/trino-filesystem/**', 'lib/trino-filesystem-*/**']
```

Comment:
thanks for fixing this

what was wrong with the previous more readable yaml list items one-per-line syntax?

Reply:
It isn't longer supported by the v5 plugin updated by @ebyhr

Reply:
so the labeler yml file does not use follow yaml syntax?

Reply:
I've followed a syntax from readme. I guess that more yaml-ish would work too (wrapping array elements on newlines)